tests/Functional/FpmRuntimeTest.php

```diff
@@ -0,0 +1,244 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Test\Functional;
+
+use GuzzleHttp\Client;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseInterface;
+
+class FpmRuntimeTest extends TestCase
+{
+    /** @var Client */
+    private $http;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->http = new Client([
+            'base_uri' => 'https://5octfcz6gc.execute-api.eu-west-1.amazonaws.com/dev/',
```

Comment:
Should we be worried about exposing this url? Could it change in time or can it get hit by a bot? Wondering if this perhaps could be an env var instead?

Reply:
It could, but I added some throttling to avoid being too exposed. I have set up billing alarms on my account, and I run these tests when creating new releases, so I am not too worried for now. TBH I think it's fine for now, I have those URLs around everywhere for demos, examples, etc.